pymrio/tools/iobalance.py

```diff
@@ -0,0 +1,189 @@
+""""
+
+A function for checking the balance of an IO system.
+
+The balance is checked by:
+1- Making sure that the matrix shapes are compatible 
+2- Checks if total inputs = total outputs
+3- Checks sector/product balance (sector/product inputs must = outputs)
+
+"""
+
+#%%
+#import packages
+
+import pymrio
+import pandas as pd
+import numpy as np
+
+
+#%%
+
+def system_is_balanced(Inter_industry: pd.DataFrame, value_added: pd.DataFrame, final_demand: pd.DataFrame, total_output:pd.DataFrame):
```

Comment:
please rename these to the variables we use in pymrio.https://pymrio.readthedocs.io/en/latest/terminology.html